bindings/ruby/README.md

```diff
@@ -0,0 +1,101 @@
+# Regorusrb
+
+**Regorus** is
+
+  - *Rego*-*Rus(t)*  - A fast, light-weight [Rego](https://www.openpolicyagent.org/docs/latest/policy-language/)
+   interpreter written in Rust.
+  - *Rigorous* - A rigorous enforcer of well-defined Rego semantics.
+
+Regorus can be used in Ruby by  Use the bundler CLI to add the gem from remote git source:
+`
+bundle add regorus --git 'https://github.com/microsoft/regorus/tree/main/bindings/ruby'
+`
+
+or manually edit your gemfile to include the following
+`
+gem "regorus", git: "https://github.com/microsoft/regorus/tree/main/bindings/ruby"
+`
+
+It is not yet available in rubygems.
+
+See [Repository](https://github.com/microsoft/regorus).
+
+To build this gem locally
+
+`rake build`
+
+then to install the gem and build the native extensions,
+
+`gem install --local ./pkg/regorusrb-0.1.0.gem`
+
+## Installation
+
+TODO: Replace `UPDATE_WITH_YOUR_GEM_NAME_IMMEDIATELY_AFTER_RELEASE_TO_RUBYGEMS_ORG` with your gem name right after releasing it to RubyGems.org. Please do not do it earlier due to security reasons. Alternatively, replace this section with instructions to install your gem from git if you don't plan to release to RubyGems.org.
```

Comment:
Not clear whom these instructions are intended for.

Reply:
Ah, I've removed this text and revised the `Installation` section header.

I've reserved the `regorus` name in Rubygems by publishing an empty v0.1.0 gem there. Whenever we're ready, we can bump versions and I can transfer ownership in Rubygems to the appropriate account(s).